### DISCUSSION TITLE
360 & Panorama Photos/Videos Player

### DISCUSSION THREAD

**Discussion by MEGAMENE** [Radiokot/photoprism-android-client#124] [Ideas]
PhotoPrism currently marks Photos and Videos as Panoramas, but doesn't have a player for them.

It would be nice if the app could support them with a proper player, maybe similar to how Google Photos does it.

**Comment by Radiokot** [comment]
It would be nice to have. If you have any photosphere or ultrawide shots, please send me them as samples.

**Reply by MEGAMENE** [reply]
I've just send a couple of sample files to your email.

**Comment by longester** [comment]
when it come true：）

**Reply by Radiokot** [reply]
Not planned so far
